labs/[1739]_scale_out_and_update_a_containerized_application_on_a_kubernetes_cluster/guide.txt

```diff
@@ -1,5 +1,5 @@
-# Configure Secure RDP using a Windows Bastion Host
-# https://www.qwiklabs.com/focuses/1737?parent=catalog
+# Build and Deploy a Docker Image to a Kubernetes Cluster
```

Comment:
Nice found,  but this must be: 
"Scale Out and Update a Containerized Application on a Kubernetes Cluster"

Reply:
I'm Sorry.
It's already fixed now :)